docs/topics/native/native-objc-interop.md

```diff
@@ -569,6 +573,56 @@ binaries.framework {
 }
 ```
 
+### Forward declarations
+
+To import forward declarations, use the `objcnames` package. For example, to import a `objcstructName` forward declaration
```

Comment:
There are no structs among Objective-C forward declarations, only classes and protocols. Within `objcnames.classes` and `objcnames.protocols` packages.

Structs are C concepts and covered by the other document and `cnames.structs` package.